Parsers/ASimUserManagement/Parsers/ASimUserManagementSentinelOne.yaml

```diff
@@ -48,22 +48,50 @@ ParserQuery: |
       5006, "GroupDeleted", "Group Deleted", "",
       5008, "GroupCreated", "User created a Manual or Pinned Group", "",
       5011, "GroupModified", "Group Policy Reverted", "Newpolicy",
+      67, "", "User 2FA Modified", "",
```

Comment:
UserModified?

Reply:
As this event indicates enable/disable 2fA activity for a user, we have defined its EventType as UserEnabled/UserDisabled.